ModConfigMenu.cs

```diff
@@ -46,8 +46,10 @@ public ModConfigMenu( bool inGame )
 
             ui.AddChild(table);
 
-            if (ingame || Constants.TargetPlatform == GamePlatform.Android)
+            if (Constants.TargetPlatform == GamePlatform.Android)
                 initializeUpperRightCloseButton();
+            else
+                upperRightCloseButton = null;
```

Comment:
I failed to fix the "half-visible" close button, so I just suppress it altogether on non-Android.